packages/model-viewer/src/features/scene-graph/material.ts

```diff
@@ -238,7 +238,11 @@ export class Material extends ThreeDOMElement implements MaterialInterface {
     const gltfMaterial = this[$sourceObject] as DefaultedMaterial;
     for (const material of this[$correlatedObjects] as
          Set<MeshStandardMaterial>) {
-      material.alphaTest = gltfMaterial.alphaCutoff;
+      if ((this[$sourceObject] as DefaultedMaterial).alphaMode === 'MASK') {
+        material.alphaTest = gltfMaterial.alphaCutoff;
+      } else {
+        material.alphaTest = 0;
```

Comment:
This feels  like a hack to me; I don't think `alphaTest` should even  be used  by the shader  if  the mode is really set to `MASK`. I'd  guess something is wrong with setting the blend mode. Have you looked at what GLTFLoader does when it gets these different materials as input?

Reply:
`material.alphaTest = 0;` was because I was paranoid, looks like it's not needed.

Reply:
Hmm, looking at three's code I think you were actually on the right track, except they base their logic on its presence: https://github.com/mrdoob/three.js/blob/dev/src/renderers/webgl/WebGLProgram.js#L637

Looks like  you need  to set it to undefined instead of zero.